arrows/klv/tests/test_klv_1107.cxx

```diff
@@ -27,30 +28,37 @@ test_read_write( klv_value const& expected_result,
   test_read_write_format< format_t >( expected_result, input_bytes );
 }
 
+// ----------------------------------------------------------------------------
+klv_lengthy< klv_imap >
+kli( double x )
+{
+  return { klv_imap{ x } };
+}
+
```

Comment:
I see this function repeated in several test_klv_###.cxx files. Is there a common header file it could be moved to?